MaeParser.cpp

```diff
@@ -21,13 +21,11 @@ static bool property_key_author_name(Buffer& buffer, char*& save);
 
 static std::string outer_block_name(Buffer& buffer);
 
-const int read_exception::BUFFER_SIZE;
-
 void read_exception::format(int line_number, int column, const char* msg)
 {
-    snprintf(m_msg, BUFFER_SIZE, "Line %d, column %d: %s\n", line_number,
-             column, msg);
-    m_msg[BUFFER_SIZE - 1] = '\0';
+    std::stringstream ss;
```

Comment:
If you have a way to keep this no-throw that'd be awesome. If you want to use a stringstream and if it throws an exception then just catch it and use the naked const char* msg as the message that'd be ok, since this should almost never happen.